R/utils-parse.r

```diff
@@ -157,6 +158,18 @@ parseFieldVisitMeasurements <- function(reportObject){
     meas_Q <- NULL
     warning("Data was retrieved for field visit measurements but it was empty. Returning NULL.")
   }
+  
+  #Check if the field visit measurements (if they exist) allow for a log axis or not, and remove zeros/negative values if indicated
+  if (!isEmptyOrBlank(meas_Q)) {
+    if(!isEmptyOrBlank(meas_Q[['value']]) && meas_Q[['value']] <= 0 && isEmptyOrBlank(excludeZeroNegativeFlag)){
+      meas_Q[['canLog']] <- FALSE
+    } else if(!isEmptyOrBlank(meas_Q[['value']]) && meas_Q[['value']] <= 0 && !isEmptyOrBlank(excludeZeroNegativeFlag) && excludeZeroNegativeFlag ) {
+      meas_Q[['canLog']] <- TRUE
+    }
+    else if(!isEmptyOrBlank(meas_Q[['value']]) && meas_Q[['value']] > 0 ) {
+      meas_Q[['canLog']] <- TRUE
+    }
```

Comment:
kind of wondering if `meas_Q[['canLog']] <- isLogged(meas_Q, isVolFlow=TRUE, excludeZeroNegativeFlag)` can be used in lieu of all these if-elses (same change could be used in `parseMinMaxIVs`). Probably out of scope, so approving!